test/fixtures/recommender.yml

```diff
@@ -0,0 +1,973 @@
+yaml_structure: regular
+sample_response:
+  - evaluationScore: 0.435
+    ontologies:
+    - acronym: EFO
+      "@id": https://data.agroportal.lirmm.fr/ontologies/EFO
+      "@type": http://data.bioontology.org/metadata/Ontology
+      links:
+        submissions: https://data.agroportal.lirmm.fr/ontologies/EFO/submissions
+        properties: https://data.agroportal.lirmm.fr/ontologies/EFO/properties
+        classes: https://data.agroportal.lirmm.fr/ontologies/EFO/classes
+        single_class: https://data.agroportal.lirmm.fr/ontologies/EFO/classes/{class_id}
+        roots: https://data.agroportal.lirmm.fr/ontologies/EFO/classes/roots
+        schemes: https://data.agroportal.lirmm.fr/ontologies/EFO/schemes
+        collections: https://data.agroportal.lirmm.fr/ontologies/EFO/collections
+        xl_labels: https://data.agroportal.lirmm.fr/ontologies/EFO/skos_xl_labels
+        instances: https://data.agroportal.lirmm.fr/ontologies/EFO/instances
+        metrics: https://data.agroportal.lirmm.fr/ontologies/EFO/metrics
+        reviews: https://data.agroportal.lirmm.fr/ontologies/EFO/reviews
+        notes: https://data.agroportal.lirmm.fr/ontologies/EFO/notes
+        groups: https://data.agroportal.lirmm.fr/ontologies/EFO/groups
+        categories: https://data.agroportal.lirmm.fr/ontologies/EFO/categories
+        latest_submission: https://data.agroportal.lirmm.fr/ontologies/EFO/latest_submission
+        projects: https://data.agroportal.lirmm.fr/ontologies/EFO/projects
+        download: https://data.agroportal.lirmm.fr/ontologies/EFO/download
+        views: https://data.agroportal.lirmm.fr/ontologies/EFO/views
+        analytics: https://data.agroportal.lirmm.fr/ontologies/EFO/analytics
+        ui: http://agroportal.lirmm.fr/ontologies/EFO
+        "@context":
```

Comment:
no need to have the `@context` and `@links`,  remove it, will make this file smaller